standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+The requirements in this section apply to all SPIFFE bundle endpoint servers and clients. Individual SPIFFE bundle endpoint profiles MAY add further requirements.
+
+### 5.2. Endpoint Profiles
+An endpoint profile describes both the transport protocol and authentication method that should be used when serving or consuming a bundle endpoint.
+
+The following sections describe the supported bundle endpoint profiles.
+
+#### 5.2.1. Web PKI (`https_web`)
+The `https_web` profile leverages publicly trusted certificate authorities to provide a low-friction path for configuring SPIFFE federation. It behaves identically to the "https" URLs that most people are familiar with when accessing web pages in their browsers. In this profile, the bundle endpoint server uses a certificate issued by a public CA, obviating the need for additional client configuration; endpoints using the `https_web` profile type are authenticated using the same public CA certificates commonly installed in modern operating systems.
+
+Please see the [Security Considerations](#7-security-considerations) section for more information about the use of public certificate authorities.
+
+##### 5.2.1.1. Endpoint URL Requirements
+Bundle endpoint URLs utilizing `https_web` MUST have the scheme set to `https` and MUST NOT include userinfo in the authority component. Other components of the URL (as defined by [RFC 3986 section 3][7]) are not constrained by this specification.
+
+For example, the URL `https://host.example.com/trust_domain` is a valid SPIFFE bundle endpoint URL for the `https_web` profile type.
+
+##### 5.2.1.2. Endpoint Parameters
+The `https_web` profile MUST NOT require any additional parameters beyond those required for every profile (namely the trust domain name, profile type, and endpoint URL) in order to function.
+
+##### 5.2.1.3. Serving a Bundle Endpoint
+SPIFFE bundle endpoint servers supporting the `https_web` transport type utilize standard TLS-protected HTTP (i.e. HTTPS). The server certificate used SHOULD be issued by a public certificate authority (as defined by the membership list of the CA/Browser forum), and MUST include the DNS name or IP address of the endpoint as an X.509 Subject Alternative Name (or Common Name). 
+
+As an interoperability concern, servers MUST NOT require client authentication to access the bundle endpoint; this includes both transport layer (e.g. client certificates) and HTTP-layer (e.g. Authentication headers) authentication schemes.
+
+Upon receiving an HTTP GET request for the correct path, the bundle endpoint server MUST respond with the most up-to-date version of the SPIFFE bundle available. The exact path value may be chosen by the operator, and appears as part of the bundle endpoint URL.
```

Comment:
and appears as *the path in* the bundle endpoint URL   ?

Reply:
or "as the path of the bundle endpoint URL" ?

Reply:
```suggestion
Upon receiving an HTTP GET request for the correct path, the bundle endpoint server MUST respond with the most up-to-date version of the SPIFFE bundle available. The choice of path is not constrained by this specification.
```

The sentence as originally written and @justinburke 's suggested corrections veer a little to far into explaining "how to URL" IMO.